reportportal_client/items/rp_base_item.py

```diff
@@ -53,8 +72,7 @@ def add_request(self, endpoint, method, request_class, *args, **kwargs):
         :param kwargs:         request object named attributes
         :return: None
         """
-        http_request = HttpRequest(method, endpoint)
         rp_request = request_class(*args, **kwargs)
-        rp_request.http_request = http_request
+        rp_request.http_request = HttpRequest(method, endpoint)
         rp_request.priority = self.weight
         self.http_requests.insert(0, rp_request)
```

Comment:
I see two ways of operating on requests and responses lists - append and insert. This may be confusing for getting the last item for each, either [0] or [-1]. Let's pick a single approach and follow it, besides appending is faster :)

Reply:
So, any thoughts on that? :)

Reply:
Sorry, didn't notice. Fixed